app/src/tasks/tasksOverviewController.js

```diff
@@ -17,6 +19,21 @@
             });
         }
         
+        function deleteTask(task){
+            var confirm = $mdDialog.confirm()
+                .title($translate.instant('COMMON.CONFIRM'))
+                .content($translate.instant('WORKFLOW.ARE_YOU_SURE_YOU_WANT_TO_DELETE_THE_TASK_AND_WORKFLOW', {task_description: task.properties.bpm_description}))
+                .ariaLabel('')
+                .targetEvent(null)
+                .ok($translate.instant('COMMON.YES'))
+                .cancel($translate.instant('COMMON.CANCEL'));
+            $mdDialog.show(confirm).then(function() {
+                workflowService.deleteWorkflow(task.workflowInstance.id).then(function(result){
+                    init();
```

Comment:
it is unclear why init() is called after delete. I suggest to rename into loadData() or something similar